src/components/Calculator.jsx

```diff
@@ -34,33 +35,26 @@ function Calculator() {
   const opreators = ['÷', 'x', '-', '+', '='];
   const { next, total, operation } = result;
   return (
-    <div
-      style={{
-        height: '100%',
-        width: '100%',
-        display: 'grid',
-        gridTemplateRows: 'repeat(6,1fr)',
-        gridTemplateColumns: 'repeat(4,1fr)',
-        gap: '.2rem',
-        padding: '.2rem',
-        background: '#ccc',
-      }}
-    >
-      <Display
-        styles={{ gridColumn: '1/5' }}
-        text={`${total || ''} ${operation || ''} ${next || ''}`}
-      />
-      {buttons.map((text) => (
-        <Button
-          key={text}
-          text={text}
-          styles={{
-            gridColumn: text === '0' ? 'span 2' : 'span 1',
-            background: opreators.includes(text) ? '#f5913e' : 'whitesmoke',
-          }}
-          onClick={() => setResult(calculate(result, text))}
+    <div className={styles.container}>
+      <h2 className={styles.caption}>Lets do some maths</h2>
+      <div className={styles.calculator}>
+        <Display
+          styles={{ gridColumn: '1/5' }}
+          text={`${total || ''} ${operation || ''} ${next || ''}`}
```

Comment:
- [ ] [OPTIONAL] nice design you got, but the background is making the calculator buttons less visible and it is not a good UI
![Screenshot from 2023-04-05 20-10-01](https://user-images.githubusercontent.com/66097371/230182337-01b218c6-67a4-488f-a769-ef0519f701f8.png)

Reply:
Thank you @VICTOR-CHUKWU I will implement your suggested changes